tests/run-make/link-arg/rmake.rs

```diff
@@ -0,0 +1,24 @@
+// In 2016, the rustc flag "-C link-arg" was introduced - it can be repeatedly used
+// to add single arguments to the linker. This test passes 2 arguments to the linker using it,
+// then checks that the compiler's output contains the arguments passed to it.
+// This ensures that the compiler successfully parses this flag.
+// See https://github.com/rust-lang/rust/pull/36574
+
+use run_make_support::rustc;
+
+fn main() {
+    let output = String::from_utf8(
+        rustc()
+            .input("empty.rs")
+            .link_arg("-lfoo")
+            .link_arg("-lbar")
```

Comment:
For some reason this is **not** failing when passed to msvc linker (link.exe), even though AFAIK this is not a valid flag syntax for link.exe. I am baffled.